html-practice/facebook-mock-up.html

```diff
@@ -0,0 +1,179 @@
+<!DOCTYPE HTML>
+<html>
+	<head>
+		<style>
+		html {
+			background-color: #e9eaed;
+			height: 100%;
+			width: 100%;
+		}
+		body {
+			width: 100%;
+			height: 800px;
+			margin: 0;
+		}
+		#Facebook-menu-bar {
+			background-color: #3a5795;
+			color: white;
+			position: relative;
```

Comment:
I think we want `position: fixed` here... Looks like that's how Facebook does it.